dask/dataframe/io/parquet.py

```diff
@@ -74,6 +74,11 @@ def _read_fastparquet(fs, paths, myopen, columns=None, filters=None,
     elif len(minmax) > 1:
         if index:
             index_col = index
+        elif index is None:
+            # attempt auto-index
+            index = pf._get_index()
+            if index:
+                index_col = index
         elif 'index' in minmax:
             index_col = 'index'
```

Comment:
It's not a roadblock for me, but I don't think there is a good argument for it being the default behavior other than "It is convenient when it happens to be what you want". I am not arguing that the feature is not useful nor to take it away, simply making it not the default. The fact that the pandas index is not a data column that can be referenced like the others is a stronger argument that this is the conservative default behavior. 

Since we now have metadata for pandas indexes, this seems less important than it was before. When interacting with "foreign" data, I think it is better to make explicit choices about the pandas data model that you're coercing to, with the default behavior being that all columns stand on equal footing in the resulting object. 

If there is no metadata indicating that a column is the pandas index, I don't think it is a hardship for users to write `infer_index=True` or `index='my_index_column'`. Presumably these are users who care about performance already, so making sure they understand the performance implications of using a column as the pandas index seems reasonable.